extensions/wikia/GlobalNavigation/templates/GlobalNavigation_searchIndex.php

```diff
@@ -14,7 +14,7 @@
 				</div>
 			</div>
 			<input id="searchInput" accesskey="f" autocomplete="off" class="search-input" name="search" placeholder="<?= Sanitizer::encodeAttribute($defaultSearchPlaceholder); ?>" type="text" value="<?= Sanitizer::encodeAttribute( $query ); ?>"/>
-			<input type="submit" id="searchSubmit" class="search-button" value="" />
+			<input type="submit" id="searchSubmit" class="search-button" value="" disabled/>
```

Comment:
If you perform a search in global navigation you are redirected to Special:Search and search input is populated with search query. With this approach button would be disabled even though value in input is set.